databases/pgpointcloud/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+
+github.setup        pgpointcloud pointcloud 1.2.5 v
+revision            0
+github.tarball_from archive
+
+name                pgpointcloud
+license             BSD-3-Clause
```

Comment:
`BSD-3-Clause` is not a valid value for this field.  The list of acceptable values can be found here: `https://trac.macports.org/wiki/PortfileRecipes#license`

Looks like it should probably be just `BSD`.

Reply:
BSD seems good to me. (According to https://opensource.org/license/bsd-3-clause/ it is also called “New BSD License”). Fixed.